### ISSUE TITLE
Not connected to any peers

### ISSUE THREAD

**Issue by subwolf** [Storj/core#124]
... but we are... I count five just on a quick peek.

````
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.140.11.246\",\"port\":4000,\"nodeID\":\"34c3c42955e989a9cc43ced91250372f99437678\",\"lastSeen\":1463989146916}","timestamp":"2016-05-23T07:39:06.916Z"}
farmer-0 {"level":"info","message":"replying to message to 16d1a8f1135f8257df3d0f77a15c4a4a096aa820","timestamp":"2016-05-23T07:39:06.928Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:06.975Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:06.979Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:06.983Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:06.987Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989146990}","timestamp":"2016-05-23T07:39:06.990Z"}
farmer-0 {"level":"info","message":"replying to message to 9b05b2a8a045b3dca88a5aea6872daeea39d5a59","timestamp":"2016-05-23T07:39:07.002Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989147046}","timestamp":"2016-05-23T07:39:07.046Z"}
farmer-0 {"level":"info","message":"replying to message to ae01b41cda7021680e74e11257e3cfaa9054855b","timestamp":"2016-05-23T07:39:07.058Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"104.196.135.20\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1463989147104}","timestamp":"2016-05-23T07:39:07.104Z"}
farmer-0 {"level":"info","message":"replying to message to 7acfa826a22ff474f8ab3650661a44bbf649f56c","timestamp":"2016-05-23T07:39:07.116Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.140.11.246\",\"port\":4000,\"nodeID\":\"34c3c42955e989a9cc43ced91250372f99437678\",\"lastSeen\":1463989147159}","timestamp":"2016-05-23T07:39:07.160Z"}
farmer-0 {"level":"info","message":"replying to message to a57f9eeba7fbe3d419d982ce07cc27d8a3d78497","timestamp":"2016-05-23T07:39:07.171Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.140.11.246\",\"port\":4000,\"nodeID\":\"34c3c42955e989a9cc43ced91250372f99437678\",\"lastSeen\":1463989147215}","timestamp":"2016-05-23T07:39:07.215Z"}
farmer-0 {"level":"info","message":"replying to message to 7aeaee81bc0cff64cf1fff4a36d7526d1262f8da","timestamp":"2016-05-23T07:39:07.227Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"104.196.135.20\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1463989147270}","timestamp":"2016-05-23T07:39:07.270Z"}
farmer-0 {"level":"info","message":"replying to message to 0ab90bd531f79ab856af124731d86d316c7c6b03","timestamp":"2016-05-23T07:39:07.281Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.248.209.126\",\"port\":4000,\"nodeID\":\"55f28065bb194ecb76bb9e609ee0096ad6cb9f13\",\"lastSeen\":1463989147360}","timestamp":"2016-05-23T07:39:07.360Z"}      [1/1944]
farmer-0 {"level":"info","message":"replying to message to 22976be4060b7e84e09662c43dbaf7b747a12fdb","timestamp":"2016-05-23T07:39:07.427Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.248.209.126\",\"port\":4000,\"nodeID\":\"55f28065bb194ecb76bb9e609ee0096ad6cb9f13\",\"lastSeen\":1463989147471}","timestamp":"2016-05-23T07:39:07.471Z"}
farmer-0 {"level":"info","message":"replying to message to 86534422c481cd5c1cb89902fe1b9086477600cd","timestamp":"2016-05-23T07:39:07.572Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"104.196.135.20\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1463989147651}","timestamp":"2016-05-23T07:39:07.652Z"}
farmer-0 {"level":"info","message":"replying to message to f88a0a4d3ec8a8626e0da07632482481384125ef","timestamp":"2016-05-23T07:39:07.652Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.140.11.246\",\"port\":4000,\"nodeID\":\"34c3c42955e989a9cc43ced91250372f99437678\",\"lastSeen\":1463989147732}","timestamp":"2016-05-23T07:39:07.732Z"}
farmer-0 {"level":"info","message":"replying to message to 8371d8b7e268f226602ded9bb35bf77e87b46bc6","timestamp":"2016-05-23T07:39:07.744Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.248.209.126\",\"port\":4000,\"nodeID\":\"55f28065bb194ecb76bb9e609ee0096ad6cb9f13\",\"lastSeen\":1463989147823}","timestamp":"2016-05-23T07:39:07.823Z"}
farmer-0 {"level":"info","message":"replying to message to 83097b8018c6707a7c0fc98f789757ab2ea37621","timestamp":"2016-05-23T07:39:07.926Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"104.196.135.20\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1463989147970}","timestamp":"2016-05-23T07:39:07.970Z"}
farmer-0 {"level":"info","message":"replying to message to 6ccac188976aecaf20116c728889b17b203ba149","timestamp":"2016-05-23T07:39:07.982Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"104.196.135.20\",\"port\":8443,\"nodeID\":\"593844dc7f0076a1aeda9a6b9788af17e67c1052\",\"lastSeen\":1463989148062}","timestamp":"2016-05-23T07:39:08.062Z"}
farmer-0 {"level":"info","message":"replying to message to cfb3641d858564faa9023b40c4c580a89b9b6adb","timestamp":"2016-05-23T07:39:08.075Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"83.248.209.126\",\"port\":4000,\"nodeID\":\"55f28065bb194ecb76bb9e609ee0096ad6cb9f13\",\"lastSeen\":1463989148119}","timestamp":"2016-05-23T07:39:08.119Z"}
farmer-0 {"level":"info","message":"replying to message to 8828191726be17e1fd138408cb1a3a0505683d9b","timestamp":"2016-05-23T07:39:08.184Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:08.232Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:08.236Z"}
farmer-0 {"level":"error","message":"Not connected to any peers","timestamp":"2016-05-23T07:39:08.240Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148243}","timestamp":"2016-05-23T07:39:08.243Z"}
farmer-0 {"level":"info","message":"replying to message to 448dec88478c6f5d7234d7eec2d6633b6b906be4","timestamp":"2016-05-23T07:39:08.348Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148392}","timestamp":"2016-05-23T07:39:08.392Z"}
farmer-0 {"level":"info","message":"replying to message to c2ee07a588cf43da288c4e88f645ff0ac3167b00","timestamp":"2016-05-23T07:39:08.404Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148448}","timestamp":"2016-05-23T07:39:08.448Z"}
farmer-0 {"level":"info","message":"replying to message to 34651c964c2bda20e5a84ca38d3aa2fa491d3a90","timestamp":"2016-05-23T07:39:08.550Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148594}","timestamp":"2016-05-23T07:39:08.594Z"}
farmer-0 {"level":"info","message":"replying to message to bcb41cfc4704e1e4c22a8cc31ee6cbea31a461bf","timestamp":"2016-05-23T07:39:08.698Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148741}","timestamp":"2016-05-23T07:39:08.741Z"}
farmer-0 {"level":"info","message":"replying to message to 725997754766c92d7b4879f364b86d570265f745","timestamp":"2016-05-23T07:39:08.841Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989148922}","timestamp":"2016-05-23T07:39:08.922Z"}
farmer-0 {"level":"info","message":"replying to message to 1ab3cc7a601ad95d35aa340dcad4cc4ad5db52f8","timestamp":"2016-05-23T07:39:09.024Z"}
farmer-0 {"level":"info","message":"received valid message from {\"protocol\":\"0.6.15\",\"address\":\"server118.web4you-gmbh.ch\",\"port\":4000,\"nodeID\":\"70b731e6be92504973d509476b2046b0422560c4\",\"lastSeen\":1463989149103}","timestamp":"2016-05-23T07:39:09.103Z"}

**Comment by gordonwritescode** [comment]
Still seeing this?

**Comment by subwolf** [comment]
Not currently but predict I will be by the morning. I think (but could be wrong) that if the seeds go offline, that message occurs. Both api.storj.io seed nodes are up and responding currently.

**Comment by gordonwritescode** [comment]
Closing as this message should be resolved in core@1.0.0 and is otherwise related to infrastructure issues, not teh codez.
